tests/base/converters/test_dataframe_to_mds.py

```diff
@@ -418,13 +418,18 @@ def test_is_json_compatible(self):
                     ]), True), True)
         ])
 
-        valid_schemas = [message_schema, prompt_response_schema, combined_schema]
+        string_map_keys_schema = StructType(
+            [StructField('map_field', MapType(StringType(), StringType()), nullable=True)])
+
+        valid_schemas = [
+            message_schema, prompt_response_schema, combined_schema, string_map_keys_schema
+        ]
 
         schema_with_binary = StructType([StructField('data', BinaryType(), nullable=True)])
 
         # Schema with MapType having non-string keys
         schema_with_non_string_map_keys = StructType(
-            [StructField('map_field', MapType(IntegerType(), StringType()), nullable=True)])
+            [StructField('map_field', MapType(BinaryType(), StringType()), nullable=True)])
```

Comment:
nit: above this line, can you change `schema_with_non_string_map_keys` variable name to `non_string_map_keys_schema` as well?